src/std/build-spec.ss

```diff
@@ -228,18 +229,17 @@
     "xml/print"
     "xml"
     ;; :std/crypto
-    (gsc: "crypto/libcrypto"
-          "-cc-options" ,(cppflags "libcrypto" "")
+    (gxc: "crypto/libcrypto" (extra-inputs: ("crypto/libcrypto-rfc5114.c"))
+          "-cc-options" ,(string-append (cppflags "libcrypto" "") " -I " srcdir)
           "-ld-options" ,(ldflags "libcrypto" "-lcrypto")
           ,@(include-gambit-sharp))
-    (static-include: "crypto/libcrypto-rfc5114.c")
```

Comment:
how is a random program supposed to find this to compile it? You _need_ this static-include directive.